AutoCollection/WebSnippet.ts

```diff
@@ -0,0 +1,128 @@
+import http = require("http");
+
+import Logging = require("../Library/Logging");
+import TelemetryClient = require("../Library/TelemetryClient");
+
+class WebSnippet {
+
+    public static INSTANCE: WebSnippet;
+
+    private static _snippet: string;
+    private static _aiUrl: string;
+    private _isEnabled: boolean;
+    private _isInitialized: boolean;
+
+    constructor(client: TelemetryClient) {
+        if (!!WebSnippet.INSTANCE) {
+            throw new Error("Web snippet injection should be configured from the applicationInsights object");
+        }
+
+        WebSnippet.INSTANCE = this;
+        WebSnippet._aiUrl = "https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js";
```

Comment:
I think it would be cool to have a config that would cause the SDK to fetch the unminified sdk and also uses the unminified version of the snippet.

Reply:
Unminified snippet is not even provided in Web SDK repo documentation, maybe loading min and unminified version of the SDK would be nice to help debugging not sure about the snippet itself, let me know what you think

Reply:
I agree it would be mainly for debugging purposes. Are the ones [here](https://github.com/microsoft/ApplicationInsights-JS/tree/master/AISKU/snippet) valid?

Reply:
Added config option